setup.cfg

```diff
@@ -30,9 +30,9 @@ ignore_errors = True
 [tox:tox]
 envlist=
     default
-    tensorflow
-    torch
-;    prophet
+;    tensorflow
```

Comment:
Why are `tensorflow` and `torch` commented out now?

Reply:
Perhaps answering my own question, is this because #538 is not merged into `feature/dep-management` yet? Perhaps I should hold off on reviewing the sub-PR's until that is merged?

Reply:
No, basically I added these all in the dep-management branch and then branched off of that. This meant the tox tests ran for all the environments in this branch which was annoying because I only really wanted to test the prophet dep bucket. Basically, as a quick fix, I just commented them out or also deleted irrelevant tests above. I'm aware that this is going to mean being very careful when merging. I think in retrospect I should have done a series of branches one of each other so: 

`dep-management -> optional-backends -> optional prophet -> optional numba`

rather than trying to isolate all the common code in the feature branch. In the end, I found myself constantly having to go back and fix something in the feature branch and then re-merging it into the work branches... The core mistake was just having code that's relevant to specific optional dependencies defined in the `dep-management` branch. 

I'm happy to update this if you think it would significantly reduce the risk of introducing errors at the point I merge the working branches into the feature branch. My current plan was to be very careful...

Reply:
I am happy with you being very careful 😅 

Assuming we still have a final opportunity to check via the PR into `master`?

Reply:
haha, yeah of course